includes/HelpCenter.php

```diff
@@ -72,7 +72,8 @@ public function newfold_help_center( \WP_Admin_Bar $admin_bar ) {
      */
     public function register_assets() {
         $asset_file = NFD_HELPCENTER_BUILD_DIR . 'index.asset.php';
-        if ( file_exists($asset_file) ) {
+        $help_enabled = true;
+        if ( file_exists($asset_file) && $help_enabled ) {
```

Comment:
@amartya-dev It seems like the two lines above may have been left behind from testing. The condition would always evaluate to true, so it looks like it isn't needed.